### PR TITLE
Update documentation with how to change config in the top level readm…

### LINKED ISSUE #802: No documentation on how to work with/edit configuration
We use expose-loader and some resolve aliases, but the latest webpacker doesn't have any documentation on how to use these with environment.js to edit the environment before toWebpackConfig()...in fact, looking at the source code this doesn't seem possible, so maybe the call toWebpackConfig() should be in environment.js and not in each separate environment so we can make config changes in one spot for global items?

### PR DESCRIPTION
**Pull request by BookOfGreg**
…e. Also reveal how to make use of internal functions

Takes comment from here: https://github.com/rails/webpacker/issues/794#issuecomment-329749982 on how to export internal functions for use.

Fixes #802

### CODE REVIEW

**Comment by gauravtiwari** [comment]
Thanks for this. Please could you move all webpack related doc to `docs/webpack.md`. You can keep the doc link though inside main README. The idea is to keep the README slimmer 😉

**Comment by BookOfGreg** [comment]
@gauravtiwari No problem. I left in the intro paragraph and link over to `webpack.md` as that's the minimum that reads well and feels appropriate. It's still quite thin this way.
Is this OK or shall I remove more of it?

File: docs/webpack.md [hunk 1]
```diff
@@ -36,6 +36,14 @@ const customConfig = require('./custom')
 module.exports = merge(environment.toWebpackConfig(), customConfig)
 ```
 
+If you need access to functions within Webpacker's configuration, you can import them like this:
```
Comment (gauravtiwari) [inline docs/webpack.md]: Shall we call it configs instead of functions?

File: README.md [hunk 2]
```diff
@@ -173,6 +174,17 @@ when running `./bin/webpack-dev-server` binstub:
...
 **Note:** Don't forget to prefix `ruby` when running these binstubs on windows
 
+### Configuration
+
+Webpacker gives you a default set of configuration files for test, development and
+production environments in `config/webpack/*.js`. You can configure each individual
+environment in their respective files or configure them all in the base
+`config/webpack/environment.js` file.
+
+By default, you shouldn't have to make any changes to `config/webpack/*.js`
```
Comment (gauravtiwari) [inline README.md]: `you shouldn't have` => `you don't need to`

**Comment by gauravtiwari** [comment]
Thanks Greg 👍  Looks good, just some minor change to wording

**Comment by BookOfGreg** [comment]
Tweaked both Readme and Webpack so they are consitent. Thanks for the speedy feedback 👍

**Comment by gauravtiwari** [comment]
Thank you. Totally missed the duplicate content between webpack and README so I have taken liberty to remove that. If you could please update TOC with doctoc, I will merge 👍

**Comment by BookOfGreg** [comment]
Done, Thank you 👍

**Comment by gauravtiwari** [comment]
thanks 🍰  🍬

**Pull request merged**